### DISCUSSION TITLE
To be able to change the intro sequnece song that plays when opening osu!lazer.

### DISCUSSION THREAD

**Discussion by ELETRICWINDGODFIST** [ppy/osu#26433] [Ideas]
You have the option to disable the osu! menu music, but when you do, it would play a random song on start up. i know it will probably be **very** low on the priority list but the option to put your favourite map when you boot up the game would be pretty cool.
![image](https://github.com/ppy/osu/assets/154603945/59955fc4-83ec-4744-a277-22fe826d8655)

**Comment by Stedoss** [comment]
https://github.com/ppy/osu/discussions/15169#discussioncomment-1490826 and kinda https://github.com/ppy/osu/discussions/15261#discussioncomment-1526482.
